app/javascript/components/weather-corner.jsx

```diff
@@ -40,8 +40,7 @@ const TempText = styled.div`
 `;
 
 const SkyIconWrapper = styled.div`
-  margin-top: 35px;
-  margin-right: 24px;
+  margin-bottom: 5px;
 `;
```

Comment:
Before this commit, the sky icon's top margin could result in its div's height growing. This just re-creates the original problem of the PR where the skyicon/temperature div was taller than the time/AMPM div.